/**:
  ros__parameters:
    # UNKNOWN, CAR, TRUCK, BUS, TRAILER, MOTORBIKE, BICYCLE, PEDESTRIAN
    passthrough_lower_bound_probability_thresholds: [0.35, 0.35, 0.35, 0.35, 0.35, 0.35, 0.35, 0.50]
    trust_distances: [50.0, 100.0, 100.0, 100.0, 100.0, 50.0, 50.0, 50.0]
    min_iou_threshold: 0.5
    use_roi_probability: false
    roi_probability_threshold: 0.5

    can_assign_matrix:
      #UNKNOWN, CAR, TRUCK, BUS, TRAILER, MOTORBIKE, BICYCLE, PEDESTRIAN <-roi_msg
      [1,       0,   0,     0,   0,       0,         0,       0,         # UNKNOWN <-detected_objects
       0,       1,   1,     1,   1,       0,         0,       0,         # CAR
       0,       1,   1,     1,   1,       0,         0,       0,         # TRUCK
       0,       1,   1,     1,   1,       0,         0,       0,         # BUS
       0,       1,   1,     1,   1,       0,         0,       0,         # TRAILER
       0,       0,   0,     0,   0,       1,         1,       1,         # MOTORBIKE
       0,       0,   0,     0,   0,       1,         1,       1,         # BICYCLE
       0,       0,   0,     0,   0,       1,         1,       1]         # PEDESTRIAN
